README.md

```diff
@@ -64,40 +82,41 @@ the pool and error is logged.
 
 Configuration is read from the yaml configuration file with the following fields:
 
-    # Listen interface and port e.g. "0:8000", "localhost:9090", ":80"
-    Listen: ":8080"
-    # List of backend URI's e.g. "http://s3.mydaracenter.org"
-    Backends:
-      - "http://s3.dc1.internal"
-      - "http://s3.dc2.internal"
-    # Limit of outgoing connections. When limit is reached, akubra will omit external backend
-    # with greatest number of stalled connections
-    ConnLimit: 100
-    # Additional not amazon specific headers proxy will add to original request
-    AdditionalResponseHeaders:
-        'Access-Control-Allow-Origin': "*"
-        'Access-Control-Allow-Credentials': "true"
-        'Access-Control-Allow-Methods': "GET, POST, OPTIONS"
-        'Access-Control-Allow-Headers': "DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type"
-    # Additional headers added to backend response
-    AdditionalRequestHeaders:
-        'Cache-Control': "public, s-maxage=600, max-age=600"
-        'X-akubra-version': '0.9.26'
-    # Read timeout on outgoing connections
-    ConnectionTimeout: "3s"
-    # Dial timeout on outgoing connections
-    ConnectionDialTimeout: "1s"
-    # Backend in maintenance mode. Akubra will skip this enpoint
-
-    # MaintainedBackend: "http://s3.dc2.internal"
-
-    # List request methods to be logged in synclog in case of backend failure
-    SyncLogMethods:
-      - PUT
-      - DELETE
-
+```yaml
+# Listen interface and port e.g. "0:8000", "localhost:9090", ":80"
+Listen: ":8080"
+# List of backend URI's e.g. "http://s3.mydaracenter.org"
+Backends:
+  - "http://s3.dc1.internal"
+  - "http://s3.dc2.internal"
+# Limit of outgoing connections. When limit is reached, Akubra will omit external backend
+# with greatest number of stalled connections
+ConnLimit: 100
+# Additional not AWS S3 specific headers proxy will add to original request
+AdditionalResponseHeaders:
+    'Access-Control-Allow-Origin': "*"
+    'Access-Control-Allow-Credentials': "true"
+    'Access-Control-Allow-Methods': "GET, POST, OPTIONS"
+    'Access-Control-Allow-Headers': "DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type"
+# Additional headers added to backend response
+AdditionalRequestHeaders:
+    'Cache-Control': "public, s-maxage=600, max-age=600"
+    'X-Akubra-Version': '0.9.26'
+# Read timeout on outgoing connections
+ConnectionTimeout: "3s"
+# Dial timeout on outgoing connections
+ConnectionDialTimeout: "1s"
+# Backend in maintenance mode. Akubra will skip this endpoint
+
+# MaintainedBackend: "http://s3.dc2.internal"
+
+# List request methods to be logged in synclog in case of backend failure
+SyncLogMethods:
+  - PUT
+  - DELETE
+```
 
 ## Limitations
 
- * Users credentials have to be identical on every backend
+ * Users credentials has to be identical on every backend
```

Comment:
Actually, it was ok as "credentials" is plural of "credential" http://www.dict.org/bin/Dict?Form=Dict2&Database=*&Query=credential 
Also, "User's credentials" (possessive instead of plural)